jest.config.mjs

```diff
@@ -0,0 +1,9 @@
+export default {
+  verbose: true,
+  testMatch: ['<rootDir>/test/**/*.?js'],
+  transform: {},
+  moduleFileExtensions: ["js", "jsx", "mjs"],
+  testPathIgnorePatterns: ["node_modules", "__mocks__", "support"],
+  roots: ["<rootDir>/src/", "<rootDir>/test/"]
```

Comment:
**annotation**:
Manually define roots to allow us to keep jest things out of the root of the project in the future.